moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -109,6 +109,25 @@ export const ProjectsListViewFiltersConf = {
         "string_is_not_null",
       ],
     },
+    {
+      name: "project_lead",
+      label: "Lead",
+      placeholder: "Project lead",
+      type: "string",
+      lookup: {
+        table_name: "moped_entity",
+        field_name: "entity_name",
+      },
+      operators: [
+        "string_contains_case_insensitive",
+        "string_begins_with_case_insensitive",
+        "string_ends_with_case_insensitive",
+        "string_equals_case_sensitive",
+        "string_does_not_equal_case_sensitive",
+        "string_is_null_special_case",
+        "string_is_not_null_special_case",
```

Comment:
this is clearly a pre-existing bug, but  the `string_is_null_special_case` ("is blank") and `string_is_not_null_special_case` ("is not blank") operators do not work correctly on this field, or on the Sponsor field. 

i'll make a separate issue.